Description: disable tests that require test assets
 These tests depend on external image files that are not distributed
 with the package. Since the data is unnecessary for building or
 using the program, the tests are disabled
Author: Nadzeya Hutsko <nadzya.info@gmail.com>
Forwarded: not-needed
Last-Update: 2025-09-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -37,7 +37,7 @@
 //!     image: StatefulProtocol,
 //! }
 //!
-//! fn main() -> Result<(), Box<dyn std::error::Error>> {
+//! /* fn main() -> Result<(), Box<dyn std::error::Error>> {
 //!     let backend = TestBackend::new(80, 30);
 //!     let mut terminal = Terminal::new(backend)?;
 //!
@@ -58,7 +58,7 @@
 //!     // It is recommended to handle the encoding result
 //!     app.image.last_encoding_result().unwrap()?;
 //!     Ok(())
-//! }
+//! } */
 //!
 //! fn ui(f: &mut Frame<'_>, app: &mut App) {
 //!     // The image widget.
